src/js/helpers/error-handler.spec.js

```diff
@@ -32,13 +37,55 @@ describe('error-handler helper', () => {
   })
 
   describe('using processWithoutFeedback()', () => {
+    it('captures Sentry message', () => {
+      const theError = new Error()
+      const sentryReportConfig = {
+        sentryReportTitle: '',
+        skipSentryReport: false,
+      }
+      sandbox.stub(ErrorHandler, 'captureSentryMessage')
+      sandbox.stub(log, 'error')
+      ErrorHandler.processWithoutFeedback(theError, sentryReportConfig)
+
+      expect(ErrorHandler.captureSentryMessage)
+        .to.have.been.calledOnceWithExactly(theError, sentryReportConfig)
+    })
+
     it('logs the error', () => {
       const theError = new Error()
+      const sentryReportConfig = {
+        sentryReportTitle: '',
+        skipSentryReport: false,
+      }
+      sandbox.stub(ErrorHandler, 'captureSentryMessage')
       sandbox.stub(log, 'error')
+      ErrorHandler.processWithoutFeedback(theError, sentryReportConfig)
 
-      ErrorHandler.processWithoutFeedback(theError)
+      expect(log.error)
+        .to.have.been.calledOnceWithExactly(theError)
+    })
+  })
+
+  describe('using captureSentryMessage()', () => {
+    it('it capture message if skipSentryReport is false', () => {
```

Comment:
1. tests only half of the functionality of the method
2. does not test branching of present `sentryReportTitle`

Reply:
add new tests